app/src/components/SoldArtworkList.tsx

```diff
@@ -37,25 +37,22 @@ class SoldArtworkList extends React.Component<ContractProps, SoldArtworkListStat
     const currentAccount = this.props.accounts[0];
     let eventsFound = 0;
     const soldInfoArray: any[] = [];
```

Comment:
is there a more specific type for this?

Reply:
Yeah there definitely is, I can change that